env.sh

```diff
@@ -45,6 +45,20 @@ action() {
         run_cmd scramv1 project CMSSW $CMSSW_VER
         run_cmd cd $CMSSW_VER/src
         run_cmd eval `scramv1 runtime -sh`
+        run_cmd git-cms-init
+        run_cmd git clone ssh://git@gitlab.cern.ch:7999/akhukhun/roccor.git RoccoR
+        run_cmd git clone git@github.com:SVfit/ClassicSVfit.git TauAnalysis/ClassicSVfit -b fastMTT_19_02_2019
+        run_cmd git clone git@github.com:SVfit/SVfitTF.git TauAnalysis/SVfitTF
+        run_cmd git clone ssh://git@gitlab.cern.ch:7999/rgerosa/particlenetstudiesrun2.git ParticleNetStudiesRun2 -b cmssw_124X
+        run_cmd mkdir -p $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
+        run_cmd scp -r ParticleNetStudiesRun2/TrainingNtupleMakerAK4/data/ParticleNetAK4 $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
+        run_cmd scp -r ParticleNetStudiesRun2/TrainingNtupleMakerAK8/data/ParticleNetAK8 $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
```

Comment:
```suggestion
        run_cmd cp -r ParticleNetStudiesRun2/TrainingNtupleMakerAK4/data/ParticleNetAK4 $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
        run_cmd cp -r ParticleNetStudiesRun2/TrainingNtupleMakerAK8/data/ParticleNetAK8 $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
```
do we really need to copy all models? I propose to copy only the latest one, to avoid big payload for crab jobs

Reply:
Do you mean if we need both AK4 and AK8, or even individual models inside those directories? I'm not sure about the latter case, I think it's safer to keep all (AK4) models for all use-cases. In the current implementation, AK8 wouldn't be needed because none of the `FatJet` branches are saved anyway after the skimming, but I thought to keep those branches if someone wants to do AK8 studies with PNet as well.

(And the usage of `scp` was again just taken from the PNet Readme instructions)

Reply:
E.g. in https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/tree/master/TrainingNtupleMakerAK4/data/ParticleNetAK4/CHS I see two networks - do we use both of them? If not - copy only the one we plan to include. If we don't use AK8 - don't copy it

Reply:
about AK8: I'm not sure if the current skimming selection is fine for AK8 studies. Probably it would required a dedicated skim. So having AK8 info in the current ntuples only increases size, without real use-case